src/platforms/javascript/common/install/cdn.mdx

```diff
@@ -8,38 +8,43 @@ import JsBundleList from "~src/components/jsBundleList";
 
 Sentry supports loading the JavaScript SDK from a CDN. Generally we suggest using our npm package (`@sentry/browser`) instead, as using the CDN can create scenarios where Sentry is unable to load due to networking issues or common extensions like ad blockers. If you _must_ use a CDN, take a look at [loading Sentry lazily with our JS loader](../lazy-load-sentry/), which provides a deferred version of our minified ES5 browser bundle. To see what other bundles are available, see [Available Bundles](#available-bundles) below.
 
+## Performance Bundle
+
+To use Sentry for error monitoring and performance tracing, you can use the following bundle:
+
 ```html {tabTitle: CDN}
 <script
-  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.min.js"
-  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.min.js', 'sha384-base64') }}"
+  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.tracing.min.js"
+  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.tracing.min.js', 'sha384-base64') }}"
   crossorigin="anonymous"
 ></script>
 ```
 
-<Alert level="info" title="Updates to naming scheme in SDK version 7">
+## Performance & Replay Bundle
 
-Version 7 of the Sentry JavaScript SDKs changed the bundles to be ES6 by default.
-Previously, the default bundles were compiled to ES5. If you need to support ES5, see [Available Bundles](#available-bundles) or the [Migration Guide to Version 7](https://github.com/getsentry/sentry-javascript/blob/master/MIGRATION.md#upgrading-from-6x-to-7x).
+To use Sentry for error monitoring, performance tracing, and Replay, you can use the following bundle:
```

Comment:
Second idea: We could link to replay here as users might not be aware of the feature (didn't try the actual links, pls double check)
```suggestion
To use Sentry for error and performance monitoring, as well as [Session Replay](platforms/javascript/session-replay/), you can use the following bundle:
```